docs/dev-concepts/faq.md

```diff
@@ -87,21 +87,29 @@ There are no message storage and retrieval-related fees incurred by developers f
 
 ### How does XMTP establish a secure and fraud-proof relationship between two identities?
 
-Blockchain accounts sign and advertise a set of keys that XMTP uses to establish a shared secret for encryption using another blockchain account’s keys. These keys attest to the authenticity of both accounts and are required to add messages to their conversation. No third-party apps or relayers are involved in this process.
+Blockchain accounts sign and advertise a set of keys to start using XMTP. XMTP uses these keys to establish a shared secret between the blockchain accounts. It then uses the shared secret to generate a key used to encrypt an invitation that allows the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
 
-To learn more about signatures, see [Sign to send and receive messages using apps built with XMTP](account-signatures).
+To learn more about these keys, see [Key generation and usage in XMTP](key-generation-and-usage).
+
+To learn more about invitation and message encryption, see [Invitation and message encryption with XMTP](invitation-and-message-encryption).
 
 ### Does each blockchain account address have a corresponding XMTP identity?
 
-Yes. Each blockchain account address is represented by an XMTP identity key. This identity key is a part of a key bundle that only that the address can use to authenticate messages.
+Yes. Each blockchain account address is represented by an XMTP identity key. This identity key is a part of a key bundle that only the address that generated it can use to authenticate messages.
+
+To learn more about XMTP identity keys, see [Key generation and usage in XMTP](key-generation-and-usage).
+
+### Why do apps built with XMTP require a user to sign with their blockchain account private keys each time they start a new messaging session?
+
+When a user starts a new messaging session, they must sign with their blockchain account private key to decrypt their XMTP key bundle. The key bundle is then used for invitation and message decryption.
 
-To learn more about XMTP identities, see [Sign to send and receive messages using apps built with XMTP](account-signatures).
+Because there is no secure place in the browser to persist a decrypted key bundle, the app can use the bundle for the current session only. Once the user starts a new session, such as after refreshing their browser, they must sign again to decrypt their key bundle.
 
-### Do apps built with XMTP need to decrypt messages with blockchain account private keys each time?
+If you are using the JavaScript client SDK, you might consider [manually handling private key storage](/docs/client-sdk/javascript/tutorials/quickstart#manually-handling-private-key-storage), but only with the understanding that this approach **requires the utmost care**.
```

Comment:
@neekolas Just opening a new comment after relocating this content. What do you think of surfacing this private key storage info here?  Is it OK - or should we keep it in the full context of the README?

Reply:
I think it's fine for now. With version 8 of the SDK, all these instructions are going to have to change. I should have more details to share soon. 

But the gist is that private keys are not always going to be accessible

Reply:
Thank you so much for taking a look, Nick - and got it - understood that the instructions will change for v8. I appreciate the heads up, as always!